app/Scenes/NIVMIZZPlayer/Player.tscn

```diff
@@ -0,0 +1,37 @@
+[gd_scene load_steps=6 format=2]
+
```

Comment:
we shouldn't have a folder called 'nivmizzplayer'. can you reconcile your changes with the existing player?